qa/tasks/cephfs/mount.py

```diff
@@ -29,11 +32,32 @@ def __init__(self, ctx, test_dir, client_id, client_remote):
         self.mountpoint_dir_name = 'mnt.{id}'.format(id=self.client_id)
         self._mountpoint = None
         self.fs = None
+        self.mounted = False
+        self._netns_name = None
+        self.nsid = -1
+        if brxnet is None:
+            self.ceph_brx_net = '192.168.0.0/16'
+        else:
+            self.ceph_brx_net = brxnet
 
         self.test_files = ['a', 'b', 'c']
 
         self.background_procs = []
 
+        # On Centos/Redhat 8 the 'brctl' has been deprecated. But the
+        # 'nmcli' in ubuntu 18.08 is buggy to setup the network bridge
+        # and there is no workaround, will continue to use the 'brctl'
+        args = ["bash", "-c",
+                "cat /etc/os-release"]
+        p = self.client_remote.run(args=args, stderr=BytesIO(),
+                                   stdout=BytesIO(), timeout=(5*60))
+        ret = re.findall(r'NAME="Ubuntu"', p.stdout.getvalue())
```

Comment:
Presumably, Ubuntu will get fixed packages eventually, and may even go the same route as RHEL8 in deprecating brctl. It might be good to look at the release version and only fall back to using brctl on older releases. Alternately we could address this when we update to newer ubuntu releases in teuthology, but that's an extra action item to take care of.

Reply:
Yeah, makes sense.
Tried this, currently only on Ubuntu 18.04 will nmcli have the problem. on 16.04 it works. I will just exclude the 18.04 and fall back to brctl.
Thanks Jeff.